exercises/list-ops/example.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+function List() {
+  return {
+    append: function (arrOne, arrTwo) {
+      var appended = [];
+
+      for (var i = 0; i < arrOne.length; i++) {
```

Comment:
Can we do this without relaying on `Array.length`?  I ask because I think the idea is to use as few built-in "list" methods as needed.  You should be able to get by with `push` and `pop` I believe.